src/image_go_nord_client/__init__.py

```diff
@@ -0,0 +1,206 @@
+import argparse
+from argparse import RawDescriptionHelpFormatter
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Optional
+
+VERSION = (Path(__file__).parent / "VERSION").read_text().strip()
+DEFAULT_EXTENSION = ".png"
+OUTPUT_IMAGE_NAME = "nord" + DEFAULT_EXTENSION
+
+__doc__ = """ImageGoNord, a converter for a rgb images to norththeme palette.
+Usage: gonord [OPTION]...
```

Comment:
Maybe better:

```suggestion
Usage: ign [OPTION]...
```

did we decided to reduce the command to something thin like `ign` in the end? Or is it still a point for discussion? I dont have a good memory and don't find where we had the discussion 😁

Reply:
Good idea, but I think this is better in another PR